doc/reportv2.Rmd

```diff
@@ -218,21 +222,14 @@ ggplotly(colourplz)
+However, because habitat can encompass a wide range of variables, it was not possible to parse out exactly what variables were important in predicting predator-prey size relationships. This was due to the manner in which environmental data was collected in this dataset. For example, the average temperature for each environment only included sea surface temperature, which was obtained from an extrinsic source (Barnes *et al.*, 2008). As a result, the temperature variable does not take into account changes in temperature with depth. Thus, for samples that were observed deep in the water column, sea surface temperature would not have been informative as a predictive variable. The same logic can be applied to other variables such as the average primary productivity of an environment, which was also measured at sea surface.
 
+On the other hand, size differences in the seasonal pack ice zone and the shelf flip when the feeding behaviour of the predator is predacious (fig. 5). Here, it is observed that shelf has a much shallower slope than both the null model and the seasonal pack ice zone, suggesting food webs regulated by predacious feeders may be more size constrained, thus producing size relationships that do not differ significantly between predators and prey. This could be a function of two different patterns. First, this pattern could be driven by the temperature-size rule. This rule states that individuals in warm environments grow more quickly due to increased metabolic rates; consequently, they reach maturity earlier, but tend to be smaller (Gibert and DeLong, 2014). In this way, size structure could be smaller in warm habitats than in cold environments like the seasonal pack ice zone. Second, this pattern may also be generated by what predacious feeders consume versus what piscivorous feeders consume. Piscivores mainly consume fish, while predacious feeders eat a more 'generalized' diet of fish, plankton, and squid. It is possible that predators with more generalized diets may be less efficient in extracting energy from their resources as their adaptations are not specialized toward a certain prey. This could reflect a tradeoff: predators with generalized diets can eat a variety of prey in their environment, but do not outperform specialists in growth and survival (Berkström *et al.*, 2014). 
```

Comment:
Would it make sense to just say that the diet is "broader/more varied" (or similar) instead of "more generalized" (not sure what a general diet is)?